constants/config.ts

```diff
@@ -2,7 +2,8 @@ import { i18n } from '../utils/lang';
 
 export const DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME = 'hubspot.config.yml';
 
-export const HUBSPOT_CONFIGURATION_FOLDER = '.hubspot';
+// Temporary change for easier debugging.
+export const HUBSPOT_CONFIGURATION_FOLDER = 'hubspot';
```

Comment:
Should this be switched back before merging?

Reply:
Just a reminder to switch this back to a dot file. I also think we may benefit from increasing the specificity here to something like `.hubspot-cli`. Otherwise it'll conflict with existing files on internal user's machines

Reply:
@kemmerle Does this need to be done as a part of this PR?